examples/grpc-client/main.go

```diff
@@ -1,3 +1,4 @@
+//go:build ignore
```

Comment:
oh these are now needed?

Reply:
Hmm,I think these were added with fmt, but we probably don't need both lines 1 & 2

Reply:
cool. let's remove them.